airflow/plugins_manager.py

```diff
@@ -183,6 +187,8 @@ def load_plugins_from_plugin_directory():
 
             for mod_attr_value in (m for m in mod.__dict__.values() if is_valid_plugin(m)):
                 plugin_instance = mod_attr_value()
+                plugin_instance.plugin_source = "airflow_plugins_directory"
+                plugin_instance.plugin_path = os.path.abspath(file_path)
```

Comment:
Please make this relative to the Airflow plugins folder path. That way you don't expose more information about the filesystem layout than is strictly necessary. 

You can do this with [`os.path.relpath()`](https://docs.python.org/3/library/os.path.html#os.path.relpath), and perhaps then prefix the path with `$AIRFLOW_HOME/plugins/` when displaying?

Reply:
makes sense. Since the plugins folder's path can be changed, we can prefix the path with `PLUGINS_FOLDER/`. I've made that change.